TradeItIosTicketSDK2/TradeItYahooEquityTradingTicketViewController.swift

```diff
@@ -416,8 +416,25 @@ class TradeItYahooEquityTradingTicketViewController: TradeItYahooViewController,
         case .orderAction:
             cell.detailTextLabel?.text = orderCapabilities.labelFor(field: .actions, value: self.order.action.rawValue)
         case .quantity:
-            let cell = cell as? TradeItNumericToggleInputCell
+            let supportedOrderQuantityTypes = orderCapabilities.supportedOrderQuantityTypes(
+                forAction: self.order.action,
+                priceType: self.order.type
+            )
+            
+            if supportedOrderQuantityTypes.isEmpty {
+                let errorResult = TradeItErrorResult(
+                    title: "Order Quantity Type Error",
+                    message: "No valid quantity type found for this action",
+                    code: TradeItErrorCode.systemError
```

Comment:
Not sure what error code this fits under, this seems the closest but let me know if you I need to change it.